### PR TITLE
Do not set links down during ssh (auto)installation

### PR DESCRIPTION
**Pull request by mchf**
bnc#944349

### CODE REVIEW

File: package/yast2-network.changes [hunk 1]
```diff
@@ -1,4 +1,11 @@
 -------------------------------------------------------------------
+Sun Nov  8 20:55:53 UTC 2015 - mfilka@suse.com
+
+- bnc#944349
+  - do not set links down when installing over network
```
Comment (mvidner) [inline package/yast2-network.changes]: The code looks good, but we should improve the explanation:

- "when installing over network" is too broad. HTTP? FTP? VNC?
- This describes *what* is done. But as we are changing a notoriously complex thing, we should say *why*.

File: src/modules/LanUdevAuto.rb [hunk 2]
```diff
@@ -205,7 +207,7 @@ def Write
       end
 
       if !rules.empty? && AllowUdevModify()
-        SetAllLinksDown()
+        SetAllLinksDown() if !Linuxrc.usessh
```
Comment (jreidinger) [inline src/modules/LanUdevAuto.rb]: so vnc is not affected?

Comment (mchf) [inline src/modules/LanUdevAuto.rb]: it is. But this is a quick fix which breaks another use case so I've limited that only on reported case.  Another improvement in this patch is a must.

Comment (mvidner) [inline src/modules/LanUdevAuto.rb]: I *guess* it is also affected but I would not mention it if we haven't *tested* it.

**Comment by mvidner** [comment]
LGTM

File: src/modules/LanUdevAuto.rb (hunk 2 above)
Comment (jreidinger) [inline src/modules/LanUdevAuto.rb]: one part is to not mention it, second one is let here bug

Comment (jreidinger) [inline src/modules/LanUdevAuto.rb]: but if release managers are fine with it, then at least please add TODO comment capturing your findings

**Pull request merged**